app/pages/organizations/organization-container.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+
+class OrganizationContainer extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      error: null,
+      fetching: false,
+      organization: null,
+      organizationAvatar: {},
```

Comment:
there are a few variable names like `organizationAvatar` and `organizationBackground`, should these simply be `avatar` or `background`? I feel until there is an actual conflict between an organization avatar and project avatar in the same module it might be better to err on the simpler side, i.e. `avatar` in this line/case.